examples/feature_convert/stl/partcad.yaml

```
@@ -0,0 +1,36 @@
+
+parts:
+  cube:
+    type: stl
+    desc: A cube defined in STL
+  cube_step:
+    type: step
+  cube_3mf:
+    type: 3mf
+  cube_brep:
+    type: brep
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)